### ISSUE TITLE
Accessibility: Tools > Marketing > Traffic > SEO textareas need accessible names.

### ISSUE THREAD

**Issue by worldomonation** [Automattic/wp-calypso#81197]
### Quick summary

Fields in the `Search engine optimization` card do not have proper accessible locators.

This is not only an issue for accessiblility users, but also for E2E tests as accessible locators do not work without names.

### Steps to reproduce

1. navigate to Tools > Marketing > Traffic.
2. open F12.
3. look at the accessibility details of the textareas and buttons under the `Search engine optimization` card.

### What you expected to happen

Accessible locators are present, like in the Jetpack version of the page (https://<site_slug>/wp-admin/admin.php?page=jetpack#/traffic).

### What actually happened

Accessible names are missing.

![image](https://github.com/Automattic/wp-calypso/assets/6549265/86e86ada-1256-417a-860e-c5ac4d028641)

### Impact

All

### Available workarounds?

There is no user impact

**Comment by worldomonation** [comment]
For reference, the Jetpack version of the page:

![image](https://github.com/Automattic/wp-calypso/assets/6549265/bf98b1b4-00e9-4bf8-8a3c-305a1853ec8d)

**Comment by cuemarie** [comment]
📌 REPRODUCTION RESULTS
- Tested on Atomic – **Replicated**

📌 FINDINGS/SCREENSHOTS/VIDEO
Replicated as described!  Thanks for the Jetpack screenshot for comparison.

![Markup on 2023-08-30 at 12:06:19](https://github.com/Automattic/wp-calypso/assets/27249804/494c8c3e-0fd6-441e-8dba-6879309a5ff0)

📌 ACTIONS
- Triage complete
